### ISSUE TITLE
[Bug]: End of string tokens in streaming mode are not filtered out.

### ISSUE THREAD

**Issue by duc-phamh** [BerriAI/litellm#391]
### What happened?

Here is my code with `stream=False`:

```
messages = [
    {
        "content": "Hello! How are you today?",
        "role": "user"
    },
]

response = litellm.completion(
    model='huggingface/meta-llama/Llama-2-7b-chat-hf', 
    messages=messages,
    api_key='api_key',
    api_base='api_base',
    max_tokens=1000,
    stream=False
)

print(response['choices'][0]['message']['content'])
```

Model's response:
```
Hello! I'm just an AI, I don't have feelings or emotions like humans do, so I don't have a physical state of being like "today". I'm here to help you with any questions or tasks you may have, so feel free to ask me anything!
```

Here is the same message with `stream=True`:

```
messages = [
    {
        "content": "Hello! How are you today?",
        "role": "user"
    },
]

response = litellm.completion(
    model='huggingface/meta-llama/Llama-2-7b-chat-hf', 
    messages=messages,
    api_key='api_key',
    api_base='api_base',
    max_tokens=1000,
    stream=True
)

for chunk in response:
    content = chunk["choices"][0].get("delta", {}).get("content")
    print(content, end='')
```

Model's response:
```
Hello! I'm just an AI, I don't have feelings or emotions like humans do, so I don't have a physical state of being like "today". I'm here to help you with any questions or tasks you may have, so feel free to ask me anything!!</s>
```

I expect the end of string token `</s>` to also be filtered out when using `stream=True`.

### Twitter / LinkedIn details

https://www.linkedin.com/in/ducpham1160/

**Comment by ghost** [comment]
hi @duc-phamh - acknowledging this issue. I'll look into this today.

**Comment by ghost** [comment]
@duc-phamh for meta-llama/llama-2 we're now checking for it's bos + eos tokens in the stream. 

Should be resolved for your scenario starting - v`0.1.697`

Please re-open this issue if it still gives you issues.

**Comment by duc-ph** [comment]
Thank you @krrishdholakia, I can see the bos + eos is now filtered out 🚀
